sw/device/lib/dif/dif_otp_ctrl_unittest.cc

```diff
@@ -385,7 +385,7 @@ TEST_F(DaiReadTest, Read32) {
   EXPECT_READ32(
       OTP_CTRL_DIRECT_ACCESS_REGWEN_REG_OFFSET,
       {{OTP_CTRL_DIRECT_ACCESS_REGWEN_DIRECT_ACCESS_REGWEN_BIT, true}});
-  EXPECT_WRITE32(OTP_CTRL_DIRECT_ACCESS_ADDRESS_REG_OFFSET, 0x620);
+  EXPECT_WRITE32(OTP_CTRL_DIRECT_ACCESS_ADDRESS_REG_OFFSET, 0x6a0);
```

Comment:
can we use the OTP_CTRL_PARAM_MANUF_STATE_OFFSET macro here? (instead of 0x6a0)?